### PR TITLE
Replace `_arg_name` with `_port_name` for Input and Output

### PR DESCRIPTION
**Pull request by lllilithyang**
# Description

In last pr, we use `_arg_name` for Input/Output and `port_name` for NodeOutput/NodeInput.
In this pr we modify code to keep the same attribute name among these classes.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by 0mza987 — APPROVED** [review]

File: sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py [hunk 1]
```diff
@@ -214,9 +214,9 @@ def __init__(
         **kwargs,
     ):
         super(Input, self).__init__(type=type)
-        # As an annotation, it is not allowed to initialize the _arg_name.
-        # The _arg_name will be updated by the annotated variable _arg_name.
-        self._arg_name = None
+        # As an annotation, it is not allowed to initialize the port_name.
+        # The port_name will be updated by the annotated variable port_name.
+        self.port_name = None
```
Comment (D-W-) [inline sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py:219]: Can we add this as private attribute, e.g. _port_name

Comment (lllilithyang) [reply sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py]: replaced

File: sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py [hunk 2]
```diff
@@ -214,9 +214,9 @@ def __init__(
         **kwargs,
     ):
         super(Input, self).__init__(type=type)
-        # As an annotation, it is not allowed to initialize the _arg_name.
-        # The _arg_name will be updated by the annotated variable _arg_name.
-        self._arg_name = None
+        # As an annotation, it is not allowed to initialize the _port_name.
+        # The _port_name will be updated by the annotated variable _port_name.
```
Comment (D-W-) [inline sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py:218]: Nit: remove?

**Review by D-W- — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

File: sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py (hunk 2 above)
Comment (lllilithyang) [reply sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py:218]: will fix in next pr

**Pull request merged**